src/components/basic/StepListItemStatic.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const StepListItemStatic = props => {
+  const { id, buttonText, children } = props;
+  return (
+    <li className="dg_step-list__list-section" id={`step-${id}`}>
+      <span className="dg_button-link dg_step-list__toggle-btn">
+        <span className="dg_step-list__toggle-btn__title">{buttonText}</span>
```

Comment:
sorry @tmccoy529 I was probably too brief in my previous comment, we still need this prop, its probably should be named something like `stepTitle` or something along those lines. If not we could have a missing title here.